test/e2e/apps/deployment.go

```diff
@@ -25,6 +25,7 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
+	apps "k8s.io/api/apps/v1beta2"
```

Comment:
hmm, nevermind... I see this is sticking with extensions/v1beta1 replicasets... when we switch this test to apps/v1beta2 ones, we might have to gate